header.php

```diff
@@ -58,13 +59,10 @@ class="collapse navbar-collapse justify-content-end"
               <a class="nav-link" href="#footer"><b>Contact Us</b></a>
             </li>
             <li class="nav-item">
-              <a class="nav-link" href="../webprogramming23_team3/login.php"><b>Login</b></a>
+              <a class="nav-link" href="login.php"><b>Login in</b></a>
```

Comment:
Remember to link the full path. Like it was before you replaced with your own. Why did you delete ../webprogramming23_team3/login.php and then just added login.php <b> login in </b> ??